src/coreclr/vm/domainassembly.h

```diff
@@ -59,13 +59,13 @@ enum NotificationStatus
 };
 
 // --------------------------------------------------------------------------------
-// DomainFile represents a file loaded (or being loaded) into an app domain.  It
+// DomainAssembly represents an assembly loaded (or being loaded) into an app domain.  It
 // is guranteed to be unique per file per app domain.
 // --------------------------------------------------------------------------------
 
-class DomainFile
+class DomainAssembly final
 {
-    VPTR_BASE_VTABLE_CLASS(DomainFile);
+    VPTR_BASE_CONCRETE_VTABLE_CLASS(DomainAssembly);
```

Comment:
Since this type is final, do we need to expose the vtable to the DAC in this way? We shouldn’t have any virtual methods if this is final.

Reply:
I have seen this pattern in other places and assumed that DAC requires a vtable for `PTR_` types. 
(i.e. a class may not be a base of anything, but has at least one virtual method, often dtor and `VPTR_BASE_CONCRETE_VTABLE_CLASS`)

As I read the docs, no-vtable classes should be ok and just need no `VPTR_`.
I will try removing this.